languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -22406,6 +22406,34 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
     <!-- ====================================================================== -->
     <category id="COLLOQUIALISMS" name="Umgangssprache" type="register">
         <!-- TODO Make one Java-rule for context-independent, colloquial expressions and maintain a list -->
+        <rulegroup id="ZUM_STATT_FUERS" name="fürs (zum) Schneiden">
+            <rule id="ZUM_STATT_FUERS1" name="fürs (zum) Schneiden1">
+                <pattern>
+                    <marker>
+                        <token>fürs<exception regexp='yes' scope='previous'>danke|danke schön</exception></token>
```

Comment:
In eine `exception` passt immer nur ein token. Wenn du sowohl "danke" als auch "danke schön" ausschließen willst, brauchst du ein Antipattern (AP).

Reply:
Wenn du eine exception oder ein AP baust, bitte immer ein example mit reinmachen, das genau das testet: Schlägt die Regel in dem angedachten Fall auch wirklich nicht an?